tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/util/K8sUtil.java

```diff
@@ -382,4 +400,16 @@ public static void assertMqttService(
                     .contains("ClusterIP");
         });
     }
+
+    private static <T extends HasMetadata> @NotNull Resource<T> loadResource(
+            final @NotNull KubernetesClient client,
+            final @NotNull String namespace,
+            final @NotNull String resourceName,
+            final @NotNull Class<T> clazz) {
+        try (final InputStream is = K8sUtil.class.getClassLoader().getResourceAsStream(resourceName)) {
+            return client.resources(clazz).inNamespace(namespace).load(is);
```

Comment:
I'm not 100% sure but this could break in a future Fabric8 version. At least in the Platform Operator the namespace doesn't seem to be applied anymore on the returned resource. I've replaced this with the following code:
```suggestion
            final var resource = client.resources(clazz).load(is);
            assertThat(resource).isNotNull();
            resource.item().getMetadata().setNamespace(namespace);
            return resource;
```